tests/cypress/support/commands/form.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+Cypress.Commands.add('createFormWithAPI', (
```

Comment:
Please add JSDoc comments for IDE recognition and documentation.

Reply:
Unless I am missing something, VScode does not support JSDoc for intellisense on custom commands.
The documentation seems to indicate that the correct way to implement IDE recognition is to add typescript definition file for each command file.

I also noticed that we are lacking the basic `jsconfig.json` file to enable cy intellisense (which I can add it in another PR).

Reply:
```js
/**
 * @memberof Cypress.Chainable.prototype
 * @method createFormWithAPI
 * @description Create a form using the GLPI API
 */
```

Adding comments like this is sufficient. If VSCode doesn't read JSDoc, that seems like a big missing feature. In PHPStorm it works OK.

What is the `jsconfig.json` file supposed to do? I don't notice any missing IDE autocomplete or documentation.

Reply:
Oh, I see. Only some IDEs like VSCode require the extra file to discover symbols...
https://docs.cypress.io/guides/tooling/IDE-integration#Reference-type-declarations-via-jsconfig

Reply:
Triple-slash is TypeScript only, no? It's a compiler-time directive.
The new commit does add autocomplete info, but that TypeScript file is doing all the work on its own. PHPStorm is smart enough to find and match `*.d.ts` files on its own.

This was discussed before (either when Vue was added or Cypress, I don't remember) that TS would add too much complexity and there aren't enough people even comfortable with vanilla JS currently.